src/src/components/properties.js

```diff
@@ -122,7 +122,7 @@ export default () => (
 
           <div>&nbsp;&nbsp;&nbsp;&nbsp;{'}'}</div>
 
-          <div>&nbsp;&nbsp;{'}'}</div>
+          <div>&nbsp;&nbsp;{'}'}'}</div>
```

Comment:
```suggestion
          <div>&nbsp;&nbsp;{'}}'}</div>
```

Otherwise it would show as `}'}` on the frontend.